systems/plants/RigidBodyManipulator.h

```diff
@@ -212,6 +212,8 @@ class DLLEXPORT_RBM RigidBodyManipulator
   DrakeCollision::ElementId addCollisionElement(const RigidBody::CollisionElement& element, const std::shared_ptr<RigidBody>& body, std::string group_name);
 
   void updateCollisionElements(const std::shared_ptr<RigidBody>& body);
+  void getTerrainPoints(int body_id, Eigen::MatrixXd &terrain_points);
```

Comment:
I was unaware of the `body_id` becoming deprecated and added the function for convenience. Will be removed.